Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -933,10 +1017,16 @@ instructionLabel(_return)
     jmp .ipint_end_ret
 
 instructionLabel(_call)
+    # First store the PC to the call site index
+    storei PC, ArgumentCountIncludingThis + TagOffset[cfr]
```

Comment:
Should we make a named macro for this instead of the comment?

Reply:
Done.